modules/image/docs/api-reference/parsed-image-api.md

```diff
@@ -0,0 +1,25 @@
+# Image Utilities
+
+A small set of image utility functions functions intended to help write image handling code that works across platforms.
+
+Background: The image returned by the [`ImageLoader`](modules/images/docs/api-reference/image-loader.md) depends on the environment, i.e. whether the application is running in a new or old browser, or under Node.js.
+
+## Usage
+
+Determine supported image types in a platform independent way:
+
+```js
+import {getDefaultImageType, isImageTypeSupported} from `@loaders.gl/images`;
+```
+
+## Functions
+
+### getDefaultImageType(): string
```

Comment:
Why is this exposed to the user? If applications implement handling different output formats, why don't they explicitly specify the type in loader options? It feels like bad coding practice to fall back to some default option that could yield something incompatible.